test/puppetlabs/puppetdb/query/reports_test.clj

```diff
@@ -97,7 +98,12 @@
     (testing "should return reports based on hash"
       (let [actual (reports-query-result :v4 ["=" "hash" report-hash])]
         (is (= (munge-expected-reports [basic])
-               (munge-actual-reports actual)))))))
+               (munge-actual-reports actual)))))
+
+    (testing "report-exists? function"
+      (is (= true (qe/object-exists? :report report-hash)))
+      (is (= false (qe/object-exists? :report "chrissyamphlett")))
+      (is (= false (qe/object-exists? :report "terrypratchett"))))))
```

Comment:
@kbarber no, repeating the test is obviously not harmful, but it's also not beneficial or necessary.